src/Common/src/CoreLib/System/Marvin.cs

```diff
@@ -5,7 +5,9 @@
 using System.Diagnostics;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
+#if CLR
 using Internal.Runtime.CompilerServices;
+#endif
 
 #if BIT64
```

Comment:
CoreFX is not built bitness specific. This ifdef won't work as expected in corefx.